ItemChanger/Logging/LoggerProxy.cs

```diff
@@ -7,7 +8,7 @@ namespace ItemChanger;
 /// </summary>
 public static class LoggerProxy
 {
-    public static ILogger Logger { get; set; } = new NullLogger();
+    public static ILogger Logger { get => ItemChangerProfile.ActiveProfileOrNull?.Logger ?? NullLogger.Instance; }
```

Comment:
This is an alright start but there definitely are cases where logging may be needed before the profile is loaded, notably the Before* lifecycle events. Out of scope here though imo.